tests/test_external_search.py

```diff
@@ -595,12 +595,12 @@ def test_query_works(self):
         # Search in publisher name.
         expect(self.moby_dick, "gutenberg")
 
-        # Title > subtitle > publisher > word found in summary
+        # Title > subtitle > word found in summary > publisher
         order = [
             self.title_match,
             self.subtitle_match,
-            self.publisher_match,
             self.summary_match,
+            self.publisher_match,
```

Comment:
This happens even though the summary match isn't a keyword match. In real usage, though, I do see publisher matches frequently showing up above summary matches -- they're just a lot closer together.